pkgs/by-name/de/dell-bios-fan-control/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  ...
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dell-bios-fan-control";
+  version = "0-unstable-2022-01-19";
+
+  src = fetchFromGitHub {
+    owner = "TomFreudenberg";
+    repo = "dell-bios-fan-control";
+    rev = "27006106595bccd6c309da4d1499f93d38903f9a";
+    hash = "sha256-3ihzvwL86c9VJDfGpbWpkOwZ7qU0E5U2UuOeCwPMR1s=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp dell-bios-fan-control $out/bin/dell-bios-fan-control
```

Comment:
could probably just use `installBin` here

Reply:
nice, didn't know that existed, thanks!